2.3.4 release Blogpost (#291)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md

title: Mixxx 2.3.4 Released
authors: Nikolaus Einhauser
status: draft
tags: 2.3, 2.3.4, release announcement
comments: yes

You know the drill, another 2.3 bugfix release. [Mixxx

> 2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.3) to be exact.

Reviewer: ```suggestion
2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.4) to be exact.
```

Author: Ah, copy-paste mistake, thank you. Fixed.

> Lets take a look at the changes:  

Reviewer 2: ```suggestion
Let's take a look at the changes:  
```

> MacOS received a bunch of love, especially in regards to possible UI performance

Reviewer 2: ```suggestion
macOS received a bunch of love, especially in regards to possible UI performance
```

https://en.wikipedia.org/wiki/MacOS

Reviewer 3: "macOS received quite some love"
too many "a bunch"s in this post ; )

Author: Yes, I'll reduce them. Bad habit :+1:

> problems. We also enabled support for MacOS 13 (Ventura), though keep in mind
> that we still don't support Apple silicon unfortunately. MacOS specific issues
> related to translations were also fixed.  

Reviewer 2: ```suggestion
problems and translation issues. We also enabled support for macOS 13 (Ventura) on Intel-CPUs, while there is still some work is in progress to support Apple silicon from Mixxx 2.4.0. 
```

> In addition, our Tango and Latenight skins received a bunch of minor tweaks

Reviewer 2: ```suggestion
In addition, our Tango and LateNight skins received a bunch of minor tweaks
```

Reviewer 3: > Thank you for taking care.

Yup, thank you!

visual fixes.  
Issues occurring during metadata import from 3rd party software were also fixed.
This includes issues when fetching data from MusicBrainz as well as importing

> metadata from Rekordbox Databases or Music Files containing Serato metadata.

Reviewer 3: ```suggestion
metadata from Rekordbox databases or music files containing Serato metadata.
```
or was there a reason for capitals?

Author: Good catch, no there really wasn't, I just didn't spot them while proofreading.

Reviewer 3: Also applies to Changelog below, but that could also be an exception because it's a title? don't care much, nit-picking..

Author: You're right, also fixed "GitHub".
